pkg/apis/devops/v1alpha1/s2ibuildertemplate_types.go

```diff
@@ -69,6 +69,13 @@ type S2iBuilderTemplateSpec struct {
 	IconPath string `json:"iconPath,omitempty"`
 }
 
+type Images struct {
```

Comment:
Calling it `ImageInfo` or something else instead of `Images` will be easier to understand;

Reply:
How about `ImagesInfo` ?